core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,27 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.reloadDummyExplorations = function(){
+          // Create a dummy exploration with random title.
+          AdminTaskManagerService.startTask();
+          $scope.setStatusMessage('Processing...');
+          for (var i = 0; i < $scope.DummyExplorationCount; ++i) {
+            var title = ['Hulk Neuroscience',
+                         'Quantum Starks',
+                         'Wonder Anatomy',
+                         'Elvish, language of "Lord of the Rings',
+                         'The Science of Superheroes'
+            ];
+            var randomTitle = Math.floor( Math.random() * title.length);
+            $http.post('/contributehandler/create_new', {
+              title: title[randomTitle],
+            }).then(function(response) {
+              $scope.setStatusMessage('Dummy explorations reloaded.');
```

Comment:
reloaded --> created

However, this seems incorrect. Don't you only want to show a success message after *all* explorations have been created?

Also, you might consider doing all this stuff in the backend. The frontend fires a request to the backend with the relevant params (e.g. num_public_explorations, num_private_explorations) and all the logic + generation is in the backend. That's more in line with the other stuff on this page. One advantage is that the backend will only return a response when it's finished generating stuff, so you have a clear "finished" signal.